Softeq.XToolkit.WhiteLabel.Droid/ViewComponents/ViewModelComponent.cs

```diff
@@ -0,0 +1,45 @@
+﻿// Developed by Softeq Development Corporation
+// http://www.softeq.com
+
+using System;
+using AndroidX.Fragment.App;
+
+namespace Softeq.XToolkit.WhiteLabel.Droid.ViewComponents
+{
+    public sealed class ViewModelComponent<TViewModel> where TViewModel : class
```

Comment:
Could you please add some info on how to use this component?

Reply:
Documentation comments were added to the class